### ISSUE TITLE
As a user I would like for the first thing I see when entering the app to be a Dashboard 

### ISSUE THREAD

**Issue by SandiOhleger** [cynerge-consulting/smokejumpers#38]
Vision for new Dashboard: 

Customized Per the user's assigned Base. 

![image](https://user-images.githubusercontent.com/82405681/114464109-4e4c3300-9bb3-11eb-832a-5f34767f630c.png)

Header instead says: "Welcome to the Missoula Smokejumper Base" 

Using the current Dashboard as an outline, we would like to utilize the statistics sited (mostly) on the Database Dashboard next to the Title and as part of a button or tile which takes you to the area. 

Example: 
![image](https://user-images.githubusercontent.com/82405681/114463079-f234df00-9bb1-11eb-9a9c-8355d17f5b7b.png)
But instead of the text shown above it should be: 
**Parachutes** _1275_ (base stat)
![image](https://user-images.githubusercontent.com/82405681/114463183-18f31580-9bb2-11eb-980e-267e29db3471.png)
So basically a combination of these two boxes above. 
Each tile can have an appropriate photo, Title /navigation and in more subtle but not smaller text a statistic that goes along with it for that base, or for the subject. 

- [ ] Customized per user's assigned Base
- [ ] Welcome to BASE NAME Smokejumper Base Header
- [ ] Link to Booster in Brief for that Base
- [ ] Link to Incidents with the current year stat for number of incidents submitted this calendar year to date. 
- [ ] Link to Parachutes with Base Stats (detailed above)
- [ ] Link to Jumpers with Base Stats
- [ ] Link to Aircraft with Base stats
- [ ] Link to Pilots with Base stats 
- [ ] Link to Reports Dashboard page

**Comment by briandavidson** [comment]
This includes changes from the way the dashboard currently functions. (like adding the specific base name, that doesn't seem to occur in current MADB).

Can we break those out into follow up issues to keep this focused on getting the dashboard to parity? Adding things beyond parity might push back our release time.

**Comment by SandiOhleger** [comment]
Since we have yet to rewrite any of the dashboards, this solution seemed the best way to include the data from the Database Dashboard and the Incidents pages that wasn't in the new workflows. If we rewrite the current dashboard, and leave out the other sub-dashboards/landing pages, they lose the stats altogether and we would be doing twice the work to rewrite the dashboard to include them. 
Tyson approved this change. It is less work that re-creating 3 dashboards. 
If the customized Welcome to BASE NAME is not practical, we can just do it the way it is now, but we should have a way to indicate the stats are per base, not nationally.
